lorien/UI/Themes/theme_dark.tres

```diff
@@ -210,7 +210,7 @@ border_width_left = 1
 border_width_top = 24
 border_width_right = 1
 border_width_bottom = 1
-border_color = Color( 0.207843, 0.211765, 0.227451, 1 )
+border_color = Color( 0.329412, 0.329412, 0.329412, 1 )
```

Comment:
I increased the brightness of the popup boundaries a bit, because otherwise overlapping windows got very confusing.

Talking about this:

![grafik](https://user-images.githubusercontent.com/542105/175888435-eefe0cf7-0d41-4291-8b31-6b3fc2d45d35.png)
(old)

![grafik](https://user-images.githubusercontent.com/542105/175888584-bbe5461d-278a-4d10-a3d1-3affb1f98dd3.png)
(New)

Reply:
Yeah makes sense. I prefer the borderless look, but i can see that that's not working with overlapping dialogs.